test/forms/SelectTransactionsFormProviderSpec.scala

```diff
@@ -251,11 +282,17 @@ class SelectTransactionsFormProviderSpec extends SpecBase {
     val startKey = "start"
     val endKey = "end"
 
-    val invalidMsgStartKey = "cf.form.error.start.date-number-invalid"
-    val invalidDateMsgStartKey = "cf.form.error.start.date.invalid.real-date"
+    val emptyMonthStartDateErrorKey = "cf.cash-account.transactions.request.start.date.empty.month"
```

Comment:
Should we wrap these messages with `messages()` for localization?

Reply:
No @Neozxc, as we are dealing only with the keys and the localization will take place in the view or in the controller (if used)